flutter/test/integrations/load_image_list_test.dart

```diff
@@ -26,8 +26,10 @@ void main() {
 
     setUp(() async {
       fixture = IntegrationTestFixture(LoadImageListIntegration.new);
-      when(fixture.binding.loadDebugImages())
+      when(fixture.binding.loadDebugImages(<String>{"0x6f80b000"}))
           .thenAnswer((_) async => imageList);
+      // when(fixture.binding.loadDebugImages(any))
+      //     .thenAnswer((_) async => imageList);
```

Comment:
@buenaflor how can I capture all `Set<String>` for `loadDebugImages`? I tried it with `any`, `any<Set<String>>()`, `captureAny` and also `argThat(isA<Set<String>>())` but nothing worked.

Reply:
The parameter of the mocked method has to be nullable, even if the parameter of the original Method is not nullable. After this change, `any` is working.